modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentNodeTranslator.java

```diff
@@ -67,7 +67,7 @@ private Contents doTranslate( final Nodes nodes, final NodesHasChildrenResult no
             }
             catch ( final Exception e )
             {
-                LOG.error( "Failed to translate node '" + node.path() + "' [" + node.id().toString() + "] to content", e );
+                LOG.debug( "Failed to translate node '" + node.path() + "' [" + node.id().toString() + "] to content", e );
```

Comment:
Maybe add special case for ContentNotFound with debug logging and other Exceptions keep logging as error?